app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -565,30 +583,28 @@ public static void markRawMediaCleanUpComplete() {
                 .edit().putBoolean(RAW_MEDIA_CLEANUP_COMPLETE, true).apply();
     }
 
-    public static boolean isPendingSyncRequestFromServer() {
+    public static boolean isPendingSyncRequestFromServerForUser() {
```

Comment:
1. This is to check if there is a pending sync for the current logged in user, it's triggered right after a successful login. I can rename it to make this more clear.
2. Confirmed.

Reply:
I understand, although my suggestion is to avoid hidden session exception if in future we call this method from other places. Supplying `username` to this method will ensure that caller of this method will handle implications of session not being available at that time.

Reply:
That's a good point